estimate-web/src/components/UserStoryCommentEditor.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from "react";
+import { PropTypes } from "prop-types";
+
+import {
+    TextField, Persona, PersonaSize, PrimaryButton, Spinner
+} from "office-ui-fabric-react";
+
+import "./UserStoryCommentEditor.less";
```

Comment:
Might be worth always putting this to the end of imports as it's a different kind of import compared to the other ones